client/state/help/reducer.js

```diff
@@ -21,9 +20,26 @@ export const selectedSiteId = createReducer( null, {
 	[ HELP_CONTACT_FORM_SITE_SELECT ]: ( state, action ) => action.siteId,
 } );
 
+/**
+ * Responsible for the help search results links
+ *
+ * @param  {Object} state  Current state
+ * @param  {Object} action Action payload
+ * @return {Object}        Updated state
+ */
+export const links = ( state = [], action ) => {
```

Comment:
Initial state should be an empty object literal `{}`? `[]` isn't likely a problem, but also isn't the shape I'd expect.

Reply:
🤦‍♂️ of course. Fixed in 8ce4da1eca8d301b1dab2465c5f3f98ba520a981